diskquota_utility.c

```diff
@@ -559,6 +546,33 @@ dq_object_access_hook(ObjectAccessType access, Oid classId,
 	}
 	LWLockRelease(diskquota_locks.extension_ddl_message_lock);
 	LWLockRelease(diskquota_locks.extension_ddl_lock);
+}
+
+/*
+ * listening on any modify on pg_extension table when:
+ * 		DROP:       will send CMD_DROP_EXTENSION to diskquota laucher
+ */
+static void
+dq_object_access_hook(ObjectAccessType access, Oid classId,
+					  Oid objectId, int subId, void *arg)
+{
+	if (classId != ExtensionRelationId)
+		goto out;
+
+	if (get_extension_oid("diskquota", true) != objectId)
+		return;
```

Comment:
```suggestion
		goto out;
```
Using `goto out;` to allow other hooks to be called